src/main/scala/com/dwolla/cloudflare/domain/model/DnsRecord.scala

```diff
@@ -1,4 +1,4 @@
-package com.dwolla.cloudflare.model
+package com.dwolla.cloudflare.domain.model
```

Comment:
It just emphasizes the classes contained in the package are a model of our domain. It's a convention I borrowed from Vaughn Vernon's book "Implementing Domain-Driven Design". See, for example, https://github.com/VaughnVernon/IDDD_Samples/tree/master/iddd_agilepm/src/main/java/com/saasovation/agilepm/domain/model.